Maximilian_express_practice/routes/admin.js

```diff
@@ -0,0 +1,26 @@
+const express = require("express");
+
+const bodyParser = require("body-parser");
+
+const app = express();
+
+const router = express.Router();
+
+const adminController = require("../controller/admin-controller");
+
+app.use(bodyParser.urlencoded({ extended: false }));
+
+
+router.get("/admin/add-product", adminController.getAdminProduct);
+router.get("/admin-product-list", adminController.getAdminProductList);
+router.post("/admin-product-list", adminController.postAdminProductList);
+router.post("/admin/add-product", adminController.postAdminproduct);
+
+router.post("/admin/product-edit", adminController.postAdminEditProduct);
```

Comment:
no need to add words like `delete` or `add` or `edit` in the route. as you are using proper HTTP methods like `post` or `get`. then it indicates what is happening.

Reply:
ok